doc/translation_process.md

```diff
@@ -44,11 +44,9 @@ Visit the [Transifex Signup](https://www.transifex.com/signup/) page to create a
 You can find the Bitcoin translation project at [https://www.transifex.com/bitcoin/bitcoin/](https://www.transifex.com/bitcoin/bitcoin/).
 
 ### Installing the Transifex client command-line tool
-The client is used to fetch updated translations. If you are having problems, or need more details, see [https://docs.transifex.com/client/installing-the-client](https://docs.transifex.com/client/installing-the-client)
+The client is used to fetch updated translations. If you are having problems, or need more details, see https://developers.transifex.com/docs/cli.
```

Comment:
I think the second sentence should be reworked to refer to https://developers.transifex.com/docs/cli unconditionally, because our docs do not provide install instructions any more.

Reply:
Agreed.